package.json

```diff
@@ -22,21 +22,21 @@
     "build-dist-dev": "webpack --config webpack.config.dev.js",
     "build-dist": "npm run clean-dist && npm run build-dist-min && npm run build-dist-dev",
     "clean-lib": "rimraf lib",
-    "build-lib": "npm run clean-lib && babel --stage 0 src -d lib",
+    "build-lib": "npm run clean-lib && babel src -d lib",
     "clean": "npm run clean-lib && npm run clean-dist",
     "build": "npm run build-lib && npm run build-dist",
-    "server-dev": "webpack-dev-server --port 3000 --config demo/webpack.config.dev.js --content-base demo",
-    "server-hot": "webpack-dev-server --port 3000 --config demo/webpack.config.hot.js --hot --content-base demo",
-    "server-test": "webpack-dev-server --port 3001 --config webpack.config.test.js",
-    "dev": "npm run server-dev & npm run server-test",
-    "hot": "npm run server-hot & npm run server-test",
+    "server-dev": "webpack-dev-server --port 3000 --config demo/webpack.config.dev.js --colors --content-base demo",
+    "server-hot": "webpack-dev-server --port 3000 --config demo/webpack.config.hot.js --colors --hot --content-base demo",
+    "server-test": "webpack-dev-server --port 3001 --config webpack.config.test.js --colors",
+    "dev": "concurrent -kr 'npm run server-dev' 'npm run server-test'",
+    "hot": "concurrent -kr 'npm run server-hot' 'npm run server-test'",
     "open-demo": "opener http://127.0.0.1:3000",
-    "open-dev": "npm run dev & npm run open-demo",
-    "open-hot": "npm run hot & npm run open-demo",
-    "lint-node": "eslint -c .eslintrc-node *.js demo/webpack.*.js",
-    "lint-react": "eslint --ext .js,.jsx -c .eslintrc-react src demo/*.jsx",
-    "lint-react-test": "eslint --ext .js,.jsx -c .eslintrc-react-test src test/client",
-    "lint": "npm run lint-node && npm run lint-react && npm run lint-react-test",
+    "open-dev": "concurrent -kr 'npm run dev' 'npm run open-demo'",
+    "open-hot": "concurrent -kr 'npm run hot' 'npm run open-demo'",
+    "lint-server": "eslint -c .eslintrc-server *.js demo/webpack.*.js",
+    "lint-client": "eslint --ext .js,.jsx -c .eslintrc-client src demo/*.jsx",
+    "lint-client-test": "eslint --ext .js,.jsx -c .eslintrc-client-test src test/client",
+    "lint": "npm run lint-server && npm run lint-client && npm run lint-client-test",
```

Comment:
Summary: use "client" and "server" instead of "react" and "node". Use `concurrent` instead of `&`.